src/server/appserver.cpp

```diff
@@ -3667,8 +3670,8 @@ ExitCode AppServer::createAndStartVfs(const Sync &sync, ExitCause &exitCause) no
         QString error;
         _vfsMap[sync.dbId()] = KDC::createVfsFromPlugin(sync.virtualFileMode(), vfsSetupParams, error);
```

Comment:
I would rather do it like this:
        auto vfsPtr = KDC::createVfsFromPlugin(sync.virtualFileMode(), vfsSetupParams, error);
        if (vfsPtr) {
             _vfsMap[sync.dbId()] = vfsPtr;
        } else { ...